sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -10,6 +10,15 @@
 
 ### Other Changes
 
+ - Updated tracing ([#29995](https://github.com/Azure/azure-sdk-for-python/pull/29995)):
+   - A span will now be created upon calls to the service that settle messages.
+   - Additional attributes added to spans:
+     - `messaging.system` - messaging system (i.e., `servicebus`)
+     - `messaging.operation` - type of operation (i.e., `publish`, `receive`, or `settle`)
+     - `messaging.batch.message_count` - number of messages sent or received (if more than one)
+     - `messaging.source.name` added to receive spans in place of `message_bus.destination`
+   - All `send` spans now contain links to `message` spans. Now, `message` spans will no longer contain a link to the `send` span.
```

Comment:
Are there any changes to any names that users should be aware of ?

Reply:
Span names stay the same. And pre-existing attribute names remain the same (at least for now). Several attributes are just added in addition. The otel plugin is intended to remap attributes to a specific set based on a specific otel schema version.